libs/@local/hash-backend-utils/src/queue/redis.ts

```diff
@@ -178,9 +171,9 @@ export class RedisQueueExclusiveConsumer implements QueueExclusiveConsumer {
     let item = await this.client.rpoplpush(processingName, processingName);
```

Comment:
I appreciate you didn't write the comment above this in this PR, but what's the actual "check" referring to in
>  `Check if there's an item which wasn't processed correctly on the last call`

I don't see anything conditional

Reply:
I gotta be honest, I have no clue what this means